From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@lodipalm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess 
Please review and approve. Thanks.

rmca14 commented:
Hi, Jennie. Can you please review and let me know if I should merge this PR? Barb is apparently OOF. Thanks.

JennieHubbard commented:
Approved!

andretms committed:
ASP.NET v1 1.0

andretms committed:
AD v1 update - 7-26

andretms committed:
Minor updates

andretms committed:
Update to Guided Setup 7-31

andretms committed:
Addressing issues from Acrolinx

andretms committed:
Fixing few issues

andretms committed:
separating notes

andretms committed:
Draft 8-1

andretms committed:
Updates 8-4

andretms committed:
Continue updates 8-14

andretms committed:
Merge branch 'master' into andret-ASPNetv1

andretms committed:
Update of content 11-16

andretms committed:
Updating to retrieve from github repo

andretms committed:
Fixing an extra comma

andretms committed:
Merge branch 'master' into andret-ASPNETv1-GuidedSetup

andretms committed:
Fixing Github sync

andretms committed:
Fixing some issues

andretms committed:
Removing duplicated code

andretms committed:
Updating screen shots

v-viinde committed:
USER STORY 1087635 SaaS App Tutorial: Edcor

Ja-Dunn committed:
edit pass: two-vm-windows

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-vm-windows

Ja-Dunn committed:
edit pass: two-vm-windows

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-vm-windows

Ja-Dunn committed:
edit pass: two-vm-windows

ShawnJackson committed:
edit pass: azure-stack-app-service-before-you-get-started

ShawnJackson committed:
edit pass: azure-stack-app-service-before-you-get-started

ShawnJackson committed:
edit pass: azure-stack-app-service-before-you-get-started

ShawnJackson committed:
edit pass: azure-stack-app-service-before-you-get-started

v-viinde committed:
Updated

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-vm-windows

andretms committed:
Merge from Master

andretms committed:
Merge branch 'master' into andret-ASPNETv1-GuidedSetup

andretms committed:
Starting changes

andretms committed:
Partial Update 0.2

Ja-Dunn committed:
edit pass: two-vm-windows

andretms committed:
Incorporating feedback, refreshing images, copying some images from v2 guided setups

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge branch 'master' into andret-ASPNETv1-GuidedSetup

andretms committed:
Updating Hyperlinks, removing and redirecting old quickstart, updated TOC

andretms committed:
Updating TOC, fixing Typo

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge branch 'master' into andret-ASPNETv1-GuidedSetup

andretms committed:
Dealing with merge conflicts

andretms committed:
Revert "Dealing with merge conflicts"

This reverts commit 87b96d6f96cd8ce09f6e5559083dfe692ff1c24e.

andretms committed:
Trying to resolve conflicts

GitHubber17 committed:
edit pass: multi-factor-authentication-whats-next

GitHubber17 committed:
edit pass: add name for bookmark

GitHubber17 committed:
edit pass: Trusted IPs procedure step spacing

v-viinde committed:
USER STORY 1143135 SaaS App Tutorial: Dealpath

v-viinde committed:
Updated

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-vm-windows

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-vm-windows

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into two-vm-windows

Ja-Dunn committed:
edit pass: two-vm-windows

andretms committed:
Excluding retired article, changing aligment of config file

andretms committed:
trying to handling conflicts

andretms committed:
Conflict handling

andretms committed:
Fixing some acrolinx issues. Trying to deal with conflicts

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge from master

andretms committed:
Dealing with conflicts

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge branch 'master' into andret-ASPNETv1-GuidedSetup

rwike77 committed:
adding graphic

rwike77 committed:
rmoving graphics

andretms committed:
Fixing issues from last reviewer

andretms committed:
Addressing additional styling issues

rwike77 committed:
updates

rwike77 committed:
updated graphyic

paulathurman committed:
edit pass: tutorial-incremental-copy-multiple-tables-powershell

v-viinde committed:
Updated

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-incremental-copy-multiple-tables-powershell

cherylmc committed:
orphaned

rwike77 committed:
text update

paulathurman committed:
edit pass: tutorial-incremental-copy-multiple-tables-powershell

cherylmc committed:
update

rwike77 committed:
updated graphic

paulathurman committed:
edit pass: tutorial-incremental-copy-multiple-tables-powershell

billmath committed:
updating

ChiragPavecha committed:
Update cloud-services-how-to-scale-portal.md

Added another information under Considerations section as customer doesn't know about it.

JasonWHowell committed:
Update connect-java.md

GitHubber17 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TFS72183

GitHubber17 committed:
edit pass: adjustments

spelluru committed:
expression updates

spelluru committed:
Update

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into expressions1213

spelluru committed:
no need of {}

v-viinde committed:
Updated

billmath committed:
updating formatting

billmath committed:
updating

billmath committed:
updating

cephalin committed:
cross link Window/Linux versions

Brenduns committed:
Storage volume work

Brenduns committed:
TOC update for Storage volume work

Brenduns committed:
TOC update for Storage volume work

Brenduns committed:
Fixing links

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge from master

Brenduns committed:
Fixing links

Brenduns committed:
Fixing links

tfitzmac committed:
added outputs section

billmath committed:
fixing indenting

v-geberr committed:
how-to review-endpoint-utterances

cherylmc committed:
Update TOC.md

carolinacmoravia committed:
C8490: Hardcoded EN locale in link

Hello, @YuriDio,

Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

v-geberr committed:
not this page so reversing

GitHubber17 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TFS72183

cephalin committed:
fix link

GitHubber17 committed:
edit pass: adjustments 2

tfitzmac committed:
updated regions and handlers

mimig1 committed:
Cosmos DB: Updating images DocDB API->SQL API #1

Brenduns committed:
updated Metadata

rwike77 committed:
more updates

mimig1 committed:
Cosmos DB: Image updates #2

Ja-Dunn committed:
Merge pull request #29558 from ShawnJackson/azure-stack-app-service-before-you-get-started

edit pass: azure-stack-app-service-before-you-get-started

mimig1 committed:
more files

PRMerger15 committed:
Merge pull request #30047 from tfitzmac/1214eg

updated regions and handlers

v-shils committed:
Merge pull request #30027 from cephalin/links

cross-link Windows/Linux versions

PRMerger12 committed:
Merge pull request #30042 from carolinacmoravia/patch-509

C8490: Hardcoded EN locale in link

ChiragPavecha committed:
Update cloud-services-how-to-scale-portal.md

making changes as suggested by Andy

Lisaco88 committed:
Merge pull request #30028 from Brenduns/storage-vm-work

Storage vm work

mimig1 committed:
Cosmos DB: Image updates #3

JeffGoldner committed:
Corrections for RP adapters; organize and update marketplace items.

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ShawnJackson committed:
Merge pull request #29702 from GitHubber17/TFS72183

edit pass: multi-factor-authentication-whats-next

mimig1 committed:
Cosmos DB: Image updates to intro.

daveba committed:
update app proxy documentation based on feedback

Lisaco88 committed:
Merge pull request #30038 from tfitzmac/1213outputs

added outputs section

mimig1 committed:
Cosmos DB: Image update #4.

carolinacmoravia committed:
C8477: Hardcoded EN locale in link

Hello, @linda33wj,

Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

PRMerger12 committed:
Merge pull request #29985 from ChiragPavecha/patch-2

Update cloud-services-how-to-scale-portal.md

mimig1 committed:
Cosmos DB: Image updates #5.

PRMerger10 committed:
Merge pull request #30056 from carolinacmoravia/patch-510

C8477: Hardcoded EN locale in link

andretms committed:
Trying to handle conflicts

ShawnJackson committed:
Update how-to-use-perfInsights.md

Lisaco88 committed:
Merge pull request #29997 from spelluru/expressions1213

expression updates

jluk committed:
fix hyperlinks and reformat overview

mimig1 committed:
update

tfitzmac committed:
deleted topics

jluk committed:
Merge branch 'master' into master

andretms committed:
Handling conflicts

spelluru committed:
value array

mimig1 committed:
Removing unused file.

spelluru committed:
acrolynx

Lisaco88 committed:
Merge pull request #30044 from v-geberr/1214-new-screenshots

1214 new screenshots

rwike77 committed:
adding graphics and links

tfitzmac committed:
fix redirection

Lisaco88 committed:
Merge pull request #30046 from mimig1/12142017images

Cosmos DB: Updating images DocDB API->SQL API #1

rwike77 committed:
removing graphic

ShawnJackson committed:
Merge pull request #29733 from Ja-Dunn/two-vm-windows

edit pass: two-vm-windows

tfitzmac committed:
fix redirect

Lisaco88 committed:
Merge pull request #30052 from mimig1/12142017imagesintro2

Cosmos DB: Image updates to intro.

Lisaco88 committed:
Merge pull request #30055 from mimig1/12142017images4

Cosmos DB: Image update #4.

rwike77 committed:
removed graphic

MGoedtel committed:
updated articles to refer to concept article

andretms committed:
Fixing issues with merge

andretms committed:
Dealing with conflicts

rwike77 committed:
updated links

andretms committed:
Again dealing with conflicts

ktoliver committed:
Merge pull request #28425 from v-viinde/edcor

USER STORY 1087635 SaaS App Tutorial: Edcor

v-shils committed:
Merge pull request #30053 from JeffGoldner/master

Latest updates to marketplace and corrections for database adapters

rwike77 committed:
updated text

rwike77 committed:
added border

ktoliver committed:
Merge pull request #30048 from mimig1/12142017images2

Cosmos DB: Image updates #2

ktoliver committed:
Merge pull request #30050 from mimig1/12142017images3

Cosmos DB: Image updates #3

ktoliver committed:
Merge pull request #29955 from cherylmc/toc

orphaned topics

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge branch 'master' into andret-ASPNETv1-GuidedSetup

spelluru committed:
fix

yujhongmicrosoft committed:
Update TOC.md

rwike77 committed:
removed prereq

JasonWHowell committed:
Update connect-java.md

Commit:
Merge pull request #30062 from spelluru/lookup1214

value array

v-shils committed:
Merge pull request #30059 from jluk/master

fix hyperlinks and reformat overview

PRMerger16 committed:
Merge pull request #30066 from MGoedtel/UpdateAgentTopics1214

updated articles to refer to concept article

ktoliver committed:
Merge pull request #29703 from v-viinde/dealpath

USER STORY 1143135 SaaS App Tutorial: Dealpath

v-shils committed:
Merge pull request #29979 from billmath/b2cusers2

updating

ktoliver committed:
Merge pull request #29826 from rwike77/vsts

Vsts

GitHubber17 committed:
Merge pull request #29631 from andretms/andret-ASPNETv1-GuidedSetup

ASP.NET Guided Setup v1

ktoliver committed:
Merge pull request #30060 from mimig1/121421017images5

Cosmos DB: Image updates #5.

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-incremental-copy-multiple-tables-powershell

rwike77 committed:
Updating portal links

billmath committed:
updating

daveba committed:
updating based on feedback

GitHubber17 committed:
Merge pull request #30061 from tfitzmac/1214author

deleted topics

PRMerger15 committed:
Merge pull request #30073 from rwike77/patch-4

Updating portal links

billmath committed:
updating

divacc committed:
Update marketplace-publishing-getting-started.md

Commit:
Merge pull request #30078 from divacc/patch-1

Update marketplace-publishing-getting-started.md

Commit:
Merge pull request #30064 from JasonWHowell/patch-3

Update connect-java.md

paulathurman committed:
edit pass: tutorial-incremental-copy-multiple-tables-powershell

PRMerger12 committed:
Merge pull request #30076 from billmath/securityupdate

updating

JasonWHowell committed:
Update connect-java.md

JasonWHowell committed:
Update connect-java.md

Ja-Dunn committed:
Merge pull request #30054 from MicrosoftDocs/app-proxy-updates

update app proxy documentation based on feedback

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

PRMerger15 committed:
Merge pull request #30082 from JasonWHowell/patch-3

Update connect-java.md

ShawnJackson committed:
Merge pull request #29970 from paulathurman/tutorial-incremental-copy-multiple-tables-powershell

edit pass: Tutorial incremental copy multiple tables powershell

v-alje committed:
Merge pull request #30086 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #30069 from yujhongmicrosoft/patch-7

Update TOC.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
